src/webots/gui/WbGuiApplication.cpp

```diff
@@ -482,13 +489,17 @@ WbSimulationState::Mode WbGuiApplication::startupModeFromPreferences() const {
 
   if (startupMode == "Real-time")
     return WbSimulationState::REALTIME;
-  if (startupMode == "Run")
-    return WbSimulationState::RUN;
   if (startupMode == "Fast")
     return WbSimulationState::FAST;
   return WbSimulationState::PAUSE;
 }
 
+bool WbGuiApplication::show3DViewFromPreferences() const {
+  WbPreferences *const prefs = WbPreferences::instance();
+  const QString show3DView(prefs->value("General/show3DView").toString());
+  return show3DView != "false";
```

Comment:
This could be probably simplified:
```suggestion
  return prefs->value("General/show3DView").toString() != "false";
```